api/leaderboard.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 const (
-	notFoundError   = "Could not find data for member"
-	defaultPageSize = 20
+	notFoundError     = "Could not find data for member"
+	defaultPageSize   = 20
+	TenantIDHeaderKey = "wildlife-platform-tenant-id"
```

Comment:
This makes podium become wildlife-specific, are we ok with that?
or should we make this an envvar to keep it opensource?